subprojects/packagefiles/tree-sitter-c/meson.build

```diff
@@ -0,0 +1,20 @@
+project('tree-sitter-c', version: 'f05e279aedde06a25801c3f2b2cc8ac17fac52ae')
```

Comment:
What do you mean by "wrong version"? It's the latest commit in https://github.com/XVilka/tree-sitter-c/commits/bare-types-support
Also, I will update it, since I plan to add more tests in the corpus. Also will move to `rizinorg/tree-sitter-c` for now.